### PR TITLE
Create gateway.yml

### PR DESCRIPTION
**Pull request by QuintenNelen**

### CODE REVIEW

File: src/gateway.yml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      line='PasswordAuthentication no'
+      state=present
+
+  - name: restart ssh
+    service:
+      name: ssh
+      state: restarted
+
+
+# kijken hoe je via ansible een cron file add dat die dan om dezoveel tijd runt
```
Comment (Jappie3) [inline src/gateway.yml:85]: maybe remove this

File: src/gateway.yml [hunk 2]
```diff
@@ -0,0 +1,85 @@
+- hosts: hostname
+  user: username
+  become: true
+
+  vars:
+    - usernames:
+      - usernames of Github account
+
+  roles:
+    - gatewayStatus
```
Comment (Jappie3) [inline src/gateway.yml:10]: did you commit this as well?

**Review by Jappie3 — CHANGES_REQUESTED** [review]
2 comments (please look into those) & 34 ansible lint errors (we'll ignore these for now)

File: src/gateway.yml (hunk 2 above)
Comment (QuintenNelen) [reply src/gateway.yml:10]: The roles are on my fork

Comment (Jappie3) [reply src/gateway.yml:10]: can you commit them to this branch (patch-2) as well?

Comment (Jappie3) [reply src/gateway.yml:10]: remove that one dutch comment in gateway.yaml and you're good to go

**Pull request merged**